appgate/openapi/openapi.py

```diff
@@ -118,7 +124,61 @@ def entity_names(
 
 
 def generate_crd(entity: Type, short_names: Dict[str, str]) -> str:
+    prev_default_object_fields = settings.default_object_fields
+
+    def attrs_fields(cls: type) -> Optional[Sequence[ObjectField]]:
+        if hasattr(cls, "__attrs_attrs__"):
```

Comment:
You can use `attrs.has` to check if it is an attrs class https://www.attrs.org/en/stable/api.html#attrs.has